### DISCUSSION TITLE
Smart Lists

### DISCUSSION THREAD

**Discussion by I7T5** [community/community#40538] [Other Feature Feedback, Questions, & Ideas]
### Product Feedback

I browse GitHub projects frequently and would like to organize my lists more logically and efficiently. It would be nice if GitHub could incorporate "Smart List" feature, sort of like smart folders on macOS, but using tags. 

Similar in spirit to #17283 (not what I want, looks too messy) and #9216, same as [pbassham's comment](https://github.com/orgs/community/discussions/9178#discussioncomment-1913484).

**Comment by ossanna16** [comment]
Hi there @I7T5 and welcome to our community! Thank you for the suggestion!
